.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+**Checklists** 
+- [ ] Added impacted package name to the issue description
+
+**Package(es) impacted by this PR:**
```

Comment:
I would remove the () notation to indicate singular vs plural. We don't need to be that grammatically accurate :). Here and in other places in this file

Also, there is an extra `e` here 

```suggestion
**Packages impacted by this PR:**
```